_partials/developer-materials-results.html.slim

```diff
@@ -41,7 +41,7 @@ div
             | Product
           td.skill Skill Level
       tbody
-        tr.material(ng-repeat="item in data.displayedMaterials" class="get-started-placeholder-{{item.fields.sys_type}}")
+        tr.material(ng-repeat="item in data.displayedMaterials | middlewareOnly" class="get-started-placeholder-{{item.fields.sys_type}}")
```

Comment:
My concern with this approach is that we would need to filter out middleware results in every place that uses the data-set. Maybe it's just one place today. However, we plan to show things like result counters in the future. 

I think it would be better to filter this at source. In particular, in the DCP query. I'll check with @lukas-vlcek about the best way to do this.